bin/deploy-websocket

```diff
@@ -33,8 +33,10 @@ assert_var description "Argument 3: DESCRIPTION is not set"
 log "Creating deployment on API Websocket Gateway"
 
 response="$(aws apigatewayv2 create-deployment --stage-name "$matrix" --description "$description-$matrix" --api-id "$api_gateway_ws_id")"
-failed_count="$(jq '.FailedEntryCount' <<<"$response")"
-if [ ! "$failed_count" = 0 ]; then
+deployment_status=$(jq -r '.DeploymentStatus' <<<"$response" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')
```

Comment:
What does piping to sed do here?

Reply:
sed to remove any trailing whitespace characters from the value obtained from jq